### PR TITLE
[REVIEW] update notebooks

### PR DESCRIPTION
**Pull request by rnyak**
This PR
-  updates `getting started with movielens` notebooks for unit tests for notebooks.
-  fixes the NVTabular bugs for `The winning-solution-recsys2020-twitter` notebook.

### CODE REVIEW

**Comment by rnyak** [comment]
@benfred, for the movielens nb unit test, Julio and I noticed that when running unit test, ETL notebook is generating rating column with nulls in the preprocessed files.  I am looking into that.

File: examples/getting-started-movielens/03b-Training-with-PyTorch.ipynb [hunk 1]
```diff
@@ -101,7 +101,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "BASE_DIR = os.path.expanduser(\"~/nvt-examples/movielens/data\")"
+    "#BASE_DIR = os.path.expanduser(\"~/nvt-examples/movielens/data\")\n",
```
Comment (benfred) [inline examples/getting-started-movielens/03b-Training-with-PyTorch.ipynb:104]: lets not just comment stuff out - and remove instead
```suggestion
```

File: examples/getting-started-movielens/01-Download-Convert.ipynb [hunk 2]
```diff
@@ -58,40 +58,20 @@
...
   },
   {
    "cell_type": "code",
-   "execution_count": 4,
+   "execution_count": 3,
    "metadata": {},
    "outputs": [],
    "source": [
-    "BASE_DIR = '~/nvt-examples/movielens/data/'"
+    "INPUT_DATA_DIR = os.environ.get('INPUT_DATA_DIR', '/model/data/')"
```
Comment (benfred) [inline examples/getting-started-movielens/01-Download-Convert.ipynb:74]: I don't love the default path being ```/model/data``` - I think we should minimally include the datasetname in the default path being used.

How about we use the previous default? :
```suggestion
    "INPUT_DATA_DIR = os.environ.get('INPUT_DATA_DIR', '~/nvt-examples/movielens/data/')"
```

File: examples/getting-started-movielens/03a-Training-with-TF.ipynb [hunk 3]
```diff
@@ -102,7 +102,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "BASE_DIR = os.path.expanduser(\"~/nvt-examples/movielens/data\")"
+    "INPUT_DATA_DIR = os.environ.get('INPUT_DATA_DIR', '/model/data/')"
```
Comment (benfred) [inline examples/getting-started-movielens/03a-Training-with-TF.ipynb:105]:
```suggestion
    "INPUT_DATA_DIR = os.environ.get('INPUT_DATA_DIR', '~/nvt-examples/movielens/data/')"
```

File: examples/getting-started-movielens/03b-Training-with-PyTorch.ipynb [hunk 4]
```diff
@@ -101,7 +101,8 @@
...
+    "INPUT_DATA_DIR = os.environ.get('INPUT_DATA_DIR', '/model/data/')"
```
Comment (benfred) [inline examples/getting-started-movielens/03b-Training-with-PyTorch.ipynb:105]:
```suggestion
    "INPUT_DATA_DIR = os.environ.get('INPUT_DATA_DIR', '~/nvt-examples/movielens/data/')"
```

**Review by benfred — COMMENTED** [review]
Thanks for this!

**Review by benfred — APPROVED** [review]

**Pull request merged**